File: copy_video.py
import shutil
import os

src = r"c:\Users\Sanju\Downloads\bdty\videoplayback (2).mp4"
dst = r"c:\Users\Sanju\Downloads\bdty\public\message_intro.mp4"

if os.path.exists(src):
    shutil.copy(src, dst)
    print(f"Copied {src} to {dst}, size: {os.path.getsize(dst)} bytes")
else:
    print(f"Source file {src} does not exist!")

File: copy_video.js
const fs = require('fs');
const path = require('path');

const src = path.join(__dirname, 'videoplayback (2).mp4');
const dst = path.join(__dirname, 'public', 'message_intro.mp4');

if (fs.existsSync(src)) {
  fs.copyFileSync(src, dst);
  console.log(`Successfully copied ${src} to ${dst}, size: ${fs.statSync(dst).size} bytes`);
} else {
  console.log(`Source file ${src} not found`);
}
